### DISCUSSION TITLE
[Script request] Listmonk

### DISCUSSION THREAD

**Discussion by mnlxr** [community-scripts/ProxmoxVE#427] [Request script]
### Application Name

Listmonk

### Website

https://listmonk.app/

### Description

Self-hosted newsletter and mailing list manager

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by bvdberg01** [comment]
I will have a look

**Reply by mnlxr** [reply]
Thanks

**Reply by bvdberg01** [reply]
> Thanks

Script is added to the repo. Can you test it?

**Reply by mnlxr** [reply]
Yes. i am working on it. it's updated and it seems fine. Thank you.

**Reply by Narkowii** [reply]
just a little thing
impossible to upload a file
i need to create manually the dir opt/listmonk/uploads/

maybe you can create automatically this dir with the installation script ?

**Reply by MickLesk** [reply]
@bvdberg01 do an PR for this. And dont forget to backup this in update script :smile:

**Reply by bvdberg01** [reply]
> just a little thing impossible to upload a file i need to create manually the dir opt/listmonk/uploads/
> 
> maybe you can create automatically this dir with the installation script ?

Tnx for the feedback!


> @bvdberg01 do an PR for this. And dont forget to backup this in update script 😄

Opened a PR.
